bundles/framework/publisher2/tools/AbstractPluginTool.js

```diff
@@ -104,6 +104,7 @@ function(sandbox, mapmodule, localization, instance, handlers) {
             me.__started = true;
         } else {
             if(me.__started === true) {
+                me.__plugin.teardownUI();
                 me.__plugin.stopPlugin(me.__sandbox);
```

Comment:
What if there is not teardownUI-function in the plugin? Shouldn't it be the plugins responsibility to remove it's UI when it's stopped?